scripts/utils/validator.ts

```diff
@@ -0,0 +1,64 @@
+const LIMITBREAK_TRANSFER_VALIDATOR_V3 =
+  '0x721C0078c2328597Ca70F5451ffF5A7B38D4E947';
+const ME_TRANSFER_VALIDATOR_V3 = '0x721C00D4FB075b22a5469e9CF2440697F729aA13';
```

Comment:
is it deployed?

Reply:
Yes, it's deployed and already used for our first ApeChain LP